ixmp/core.py

```diff
@@ -1380,6 +1382,10 @@ def solve(self, model, case=None, model_file=None, in_file=None,
             Method to execute arbitrary non-model code. Must accept a single
             argument, the Scenario. Must return a non-:obj:`False` value to
             indicate convergence.
+        gams_args : list of str, optional
+            additional arguments for the CLI call to gams
```

Comment:
1. Capitalize 'Additional' and 'GAMS', add a period at the end of the description.
2. Add one blank line before the bullet list, as required by ReST, and a period at the end of the bullet.
3. Can you please add a link to the GAMS documentation?